ui_test.go

```diff
@@ -171,6 +161,16 @@ func TestPrefixedUiWarn(t *testing.T) {
 	}
 }
 
+func TestAdvancedUi_Print(t *testing.T) {
+	writer := new(bytes.Buffer)
+	ui := &BasicUi{Writer: writer}
+	ui.WriteString("HELLO")
+
+	if writer.String() != "HELLO" {
+		t.Fatalf("bad: %s", writer.String())
+	}
+}
```

Comment:
Ah!  Can you rename that to `TestAdvancedUi_PrintPrefix` or something?  Not a big deal, but it'd be nice to be consistent for `go test -run TestAdvancedUi` purposes.